html-css/week1/index.html

```diff
@@ -0,0 +1,185 @@
+                </div>
+                <div id="test-item">
+                    <h2>Xénia</h2>
+                    <h2>Partner, Noje be</h2>
+                    <p>"Very nice professional encounter with Brice. Even separated by a few kilometers because we are in Belgium, Brice has always been very responsive and proactive. Always attentive to ideas, he perfectly creates a beautiful visual identity. I highly recommend his services for creating websites, business cards, flyers, etc. (We only have praises for your work)."</p>
+                </div>
+                <div id="test-item">
+                    <h2>Riaz</h2>
+                    <h2>CEO, Dealrun</h2>
+                    <p>""Brice is a person who was attentive and delivered a result on time, the result was top-notch, Brice's work brought a fresh look to my website's design. I recommend."</p>
+                </div>
+            </div>
+        </section>
+        <section id="aboutme" class="my-introduction">
+            <h1>Sayed Jalil Hussaini</h1>
+            <h3>Vejle</h3>
+            <h2>I design and develop websites that are both elegant, intuitive, and accessible.</h2>
+            <p>Driven by a deep passion for graphic design and web development, I specialize in harmonizing the logic of code with the aesthetics of design. This ensures effective intervention on all aspects of the project, without intermediaries.</p>
+            <h2>I also specialize in creating your brand image: logo, banner, and much more.</h2>
+            <p>From a simple idea, a unique brand identity and an exceptional website are created. My clients appreciate my versatility and the quality of my work, forged by significant experience in an agency.</p>
+            <a href="#">Discover my journey(LinkedIn).</a>
+            <div class="skill-items">
+                <div class="graphic">
+                    <h2>.Graphic and UX/UI design</h2>
+                    <span> <i class="fa-regular fa-image"></i></span>
+                    <span><i class="fa-solid fa-compass-drafting"></i></span>
+                    <span><i class="fa-solid fa-paint-roller"></i></span>
+                    <span><i class="fa-solid fa-pen"></i></span>
+                </div>
+                <div class="web">
+                    <h2>.Web development</h2>
+                    <span> <i class="fa-brands fa-js"></i></span>
+                    <span><i class="fa-brands fa-react"></i></span>
+                    <span><i class="fa-brands fa-node-js"></i></span>
+                    <span><i class="fa-brands fa-angular"></i></span>
+                </div>
+            </div>
+        </section>
+
+   </main>
+   <footer>
+    © Jalil Hu      <a href="#">Privacy Policy</a> | <a href="#">Legal Notice</a>
+   </footer>
+    <script>
+        function scrollToAboutme() {
```

Comment:
Already some JavaScript making its way into the code! 😮 In this case, we could accomplish the same using pure CSS, can you figure out how this would work? (hint: look into `scroll-behavior`)

Reply:
<a> tag and with href property that navigates to give id